server/models/Role.js

```diff
@@ -0,0 +1,17 @@
+// https://sequelize.org/master/manual/model-basics.html
+export default (sequelize, DataTypes) => {
+  const Role = sequelize.define(
+    'Role',
+    {
+      name: { type: DataTypes.STRING, allowNull: false, unique: true },
+      permissions: DataTypes.STRING,
+    },
+    {
+      tableName: 'roles',
+    },
+  );
+  Role.associate = () => {
```

Comment:
@fordarnold There's a need to define associations on both role & user models.

Reply:
Refer to previous reply: https://github.com/STACK-UP-3/warriorz-backend/pull/19#discussion_r409862923